code/lab/lab.cpp

```diff
@@ -160,6 +170,25 @@ void rotate_view(int dx, int dy)
 	CLAMP(lab_cam_phi, 0.01f, PI - 0.01f);
 }
 
+void reset_view()
+{
+	//reset position
+	Lab_model_pos = ZERO_VECTOR;
```

Comment:
MSVC 2015 doesn't like this line since it has some kind of bug with nested initializer lists. Try using `vmd_zero_vector` instead.